opentech/apply/funds/templates/funds/includes/delegated_form_base.html

```diff
@@ -2,23 +2,7 @@
 <form class="form {{extra_classes}}" method="post" id="{{ form.name }}">
     {% csrf_token %}
     <div class="form__item">
-        {{ form.non_field_errors }}
-
-        {% for field in form %}
-            {{ field.errors }}
-
-            {# TODO: Add styles around icon later #}
-            {% if field.field.widget.attrs.role and field.field.widget.attrs.role.icon %}
-                {% image field.field.widget.attrs.role.icon max-20x20 %}
-            {% endif %}
-
-
-            {{ field.label_tag }} {{ field }}
-
-            {% if field.help_text %}
-                {{ field.help_text|safe }}
-            {% endif %}
-        {% endfor %}
+        {{ form }}
```

Comment:
Now that we have moved the icon out to the widget template, I don't think we need to load wagtailimages_tags.